v19.2/cockroachcloud-cluster-management.md

```diff
@@ -7,6 +7,31 @@ build_for: [cockroachcloud]
 
 This page describes the cluster management, backup/restore, and cluster deletion workflows.
 
+## View Clusters page
+
+On [signing in to the Console](cockroachcloud-create-your-account.html#sign-in), the **Clusters** page is displayed. The **Clusters** page provides a high-level view of your clusters.
+
+For each cluster, the following details display:
+
+- The cluster's **Name**
+- The **Version** of CockroachDB the cluster is running
+- The **Regions** the cluster is in
+- The date and time the cluster was **Created**
+
+To view and manage a specific cluster, click the name of the cluster. The [**Overview**](#view-cluster-overview) page will display.
+
+## Access tab
+
+On the **Clusters > Access** tab, Console administrators can manage Console access settings for the organization.
+
+## View cluster overview
+
+The **Cluster Overview** page displays a list of the selected cluster's nodes.
+
+For each node, the page displays the node's `Name`, nested under its region.
```

Comment:
Nit: Without an image, I'm slightly confused by this description. It sounds like both the `Name` and region are displayed for the node, but this makes it sound as though the `Name` is the only thing displayed. (Maybe it makes sense with an image, so take this with a grain of salt.)